src/world/map.c

```diff
@@ -750,6 +750,10 @@ void game_command_remove_scenery(int* eax, int* ebx, int* ecx, int* edx, int* es
 
 	bool sceneryFound = false;
 	rct_map_element* map_element = map_get_first_element_at(x / 32, y / 32);
```

Comment:
We assume every tile to have a surface element, and even create one if it doesn't exist when we load a park. I don't think there is any point doing a check here and no where else... unless of course the x and y position is out of bounds. In which case that is a problem for most game commands.